README.md

```diff
@@ -239,6 +241,9 @@ Timeout in milliseconds between each retry if a token refresh should fail. Defau
 **enablePkce** `<Boolean>` (optional)
 Enables PKCE mechanism to provide additional protection during code to token exchanges. Default is `false`.
 
+**unauthorizedRequestRedirectTimeout** `<Number>` (optional)
+Debounce timeout for redirection after (multiple) `401` responses are received. Set to `0` for disabling debouncing the handler. Default is `1000`.
```

Comment:
```suggestion
Debounce timeout for redirection after (multiple) `401` responses are received to prevent redirect loops (at the cost of a small delay). Set to `0` to disable debouncing. Default is `1000`.
```